test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs

```diff
@@ -30,24 +31,18 @@ protected ApplicationTestFixture(string applicationName)
 
         public ILogger Logger { get; private set; }
 
-        public DeploymentResult CreateDeployment()
-        {
-            lock (_deploymentLock)
```

Comment:
Since each test is calling `CreateDeploymentAsync`, they would have a separate output folder for each of them. This is how it should have been before my change.